Kube/scripts/cluster-min.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
@BenyH are we sure we want to have this change for everyone ?

Reply:
it's not for everyone - by default it is the same (1 CPU)
only if you use the -m option it will use it